Docs: apply basic style guide fixes: part 2 (#43742)
home-assistant/home-assistant.github.io: source/_docs/frontend/icons.markdown

Being open-source like Home Assistant, the material design icons library is always accepting suggestions and contributions to expand the library.

{% note %}

> Before suggesting or creating a new icon, it is very important that you [search the current library](https://pictogrammers.com/library/mdi/) and [search all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue), open and closed, on their GitHub. Try searching with different terms that might mean the same thing. such as "user," "person," or "account."

Author: ```suggestion
Before suggesting or creating a new icon, it is very important that you [search the current library](https://pictogrammers.com/library/mdi/) and [search all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue), open and closed, on their GitHub. Try searching with different terms that might mean the same thing. For example: "user," "person," or "account."
```

> Before suggesting or creating a new icon, it is very important that you [search the current library](https://pictogrammers.com/library/mdi/) and [search all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue), open and closed, on their GitHub. Try searching with different terms that might mean the same thing. For example: "user," "person," or "account."

Reviewer: ```suggestion
Before suggesting or creating a new icon, it is very important that you [search the current library](https://pictogrammers.com/library/mdi/) and [search all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue), open and closed, on their GitHub. Try searching with different terms that might mean the same thing. For example: "user", "person", or "account".
```

Author: Thanks for the catch!
